src/sonic-yang-models/tests/files/sample_config_db.json

```diff
@@ -252,258 +254,295 @@
                 "alias": "Eth1/4",
                 "lanes": "68",
                 "description": "",
                 "speed": "11100",
+                "tpid": "0x88A8",
                 "admin_status": "up"
             },
             "Ethernet4": {
                 "alias": "Eth2/1",
                 "lanes": "69",
                 "description": "",
                 "speed": "11100",
+                "tpid": "0x9100",
                 "admin_status": "up"
             },
             "Ethernet5": {
                 "alias": "Eth2/2",
                 "lanes": "70",
                 "description": "",
                 "speed": "11100",
+                "tpid": "0x9200",
                 "admin_status": "up"
             },
             "Ethernet6": {
                 "alias": "Eth2/3",
                 "lanes": "71",
                 "description": "",
                 "speed": "11100",
+                "tpid": "0x8100",
                 "admin_status": "up"
             },
             "Ethernet7": {
                 "alias": "Eth2/4",
                 "lanes": "72",
                 "description": "",
                 "speed": "11100",
+                "tpid": "0x8100",
                 "admin_status": "up"
             },
             "Ethernet8": {
                 "alias": "Eth3/1",
                 "lanes": "73",
                 "description": "",
                 "speed": "11100",
+                "tpid": "0x8100",
                 "admin_status": "up"
             },
             "Ethernet9": {
                 "alias": "Eth3/2",
                 "lanes": "74",
                 "description": "",
                 "speed": "11100",
+                "tpid": "0x8100",
                 "admin_status": "up"
             },
             "Ethernet10": {
                 "alias": "Eth3/3",
                 "lanes": "75",
                 "description": "",
                 "speed": "11100",
+                "tpid": "0x8100",
                 "admin_status": "up"
             },
             "Ethernet11": {
                 "alias": "Eth3/4",
                 "lanes": "76",
                 "description": "",
                 "speed": "11100",
+                "tpid": "0x8100",
                 "admin_status": "up"
             },
             "Ethernet12": {
                 "alias": "Eth4/1",
                 "lanes": "77",
                 "description": "",
                 "speed": "11100",
+                "tpid": "0x8100",
                 "admin_status": "up"
             },
             "Ethernet13": {
                 "alias": "Eth4/2",
                 "lanes": "78",
                 "description": "",
                 "speed": "11100",
+                "tpid": "0x8100",
                 "admin_status": "up"
             },
             "Ethernet14": {
                 "alias": "Eth4/3",
                 "lanes": "79",
                 "description": "",
                 "speed": "11100",
+                "tpid": "0x8100",
```

Comment:
We do not need to add it in all ports, just add it in 1 or 2 ports.

Reply:
Going forward after this PR is approved and merged, the config DB when generated from the Minigraph will always have TPID 0x8100.  It is similar to the MTU handling where there is a default value set in config DB.
I like to keep this change to set TPID attribute for all the interfaces (port and LAG). this way it is more in sync when user inspect config DB.